lib/browser/api/base-window.ts

```diff
@@ -43,7 +43,7 @@ Object.defineProperty(BaseWindow.prototype, 'kiosk', {
 });
 
 Object.defineProperty(BaseWindow.prototype, 'documentEdited', {
-  get: function () { return this.isFullscreen(); },
+  get: function () { return this.isFullScreen(); },
```

Comment:
This never should have been isFullScreen.  isDocumentEdited is the proper function to call
```suggestion
  get: function () { return this.isDocumentEdited(); },
```

Reply:
This makes way more sense :) I thought for some reason this was intentional